addons/web/static/src/views/calendar/calendar_controller.js

```diff
@@ -15,7 +15,7 @@ import { CalendarQuickCreate } from "./quick_create/calendar_quick_create";
 
 import { Component, useState } from "@odoo/owl";
 
-const SCALE_LABELS = {
+export const SCALE_LABELS = {
```

Comment:
weird diff in this commit

Reply:
it is used by the calendar editor, but has nothing to do with the changes of this commit -> moved in another one